packages/headless/src/features/commerce/facets/facet-set/facet-set-slice.ts

```diff
@@ -79,10 +80,18 @@ export const commerceFacetSetReducer = createReducer(
     builder
       .addCase(fetchProductListing.fulfilled, handleQueryFulfilled)
       .addCase(executeSearch.fulfilled, handleQueryFulfilled)
-      .addCase(
-        executeCommerceFieldSuggest.fulfilled,
-        handleFieldSuggestionsFulfilled
+      .addCase(executeCommerceFieldSuggest.fulfilled, (state, action) =>
+        handleFieldSuggestionsFulfilled(state, action.payload.facetId)
       )
+      .addCase(fetchQuerySuggestions.fulfilled, (state, action) => {
```

Comment:
We should cover this new case in facet-set-slice.test.ts!

Reply:
Done in https://github.com/coveo/ui-kit/pull/4184/commits/37e44fa6a05d02f7fee52a9901239c6d52aad157